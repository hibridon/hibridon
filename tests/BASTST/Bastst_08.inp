     BASTST       
   8                             ibasty
  F  F  F  T                     logdfl, airyfl, readpt, bastst
    1.0    1.0     0.1           rstart, rendld, spac
  F                              prairy
  1.150      3.00    25.0 15.00  tolai, rincr, rendai, fstfac
   1                             nerg
   5000.0
   16.47000                      xmu
   30.0000                       rcut
  20  20   5   1   0   0   1     jtot1,jtot2,jtotd,jlpar,numin,numax,nud
  48   2                         lscreen, iprint
    3    1                       nnout,niout
    00  11 02  
    0
  F  F  F  F  F                  prlogd, prsmat, prt2, t2test, wrsmat
  F  F  F  F  F                  wrpart, prpart, prxsec, wrxsec, wavefl
  F  T  F  F  F                  noprin, chlist, ipos, nucros, photof
  F  F  F  F  F                  flaghf, csflag, flagsu, ihomo, twomol
  F  F                           rsflag, boundc
  1 1 3                          nterm, nsym, numj
  0.8 0.002 0.0002               brot, drot, hrot
  00 11 02                       j1j2
nofile.dat