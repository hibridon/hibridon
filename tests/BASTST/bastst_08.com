inp=bastst_08.inp
out=bastst_08.out
run
save=save_08.inp