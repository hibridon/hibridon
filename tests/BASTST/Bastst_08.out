      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   2
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** 1SIG+1SI system parameters:
 NTERM  =   1        NSYM   =   1
 BROT   = 0.8000     DROT   = 2.0000E-03 HROT   = 2.0000E-04
 LAMMIN:    2
 LAMMAX:    4
 MPROJ:     0
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= F
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT:  3; JOUT:  0  11   2
 INDOUT:    0
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-01-24 at 11:46
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: 490a05d5957aa87d9c9668b1fd83ca5b6dbd246d
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Jan 24 12:01:36 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500

   **  CC  HF-HF  **  E= 5000.000  JTOT=  20  SYM= 1  PAR= 1  N-LAM= 1

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=   30.00
 **  LIST OF ROTATIONAL LEVELS :
   N   J1  J2   E-(CM-1)

   1    0   0    0.000
   2    1   1    3.187
   3    0   2    4.771
 CHECK OF VLMTWO:  JLPAR =  1, NSYM =  1
 IROW  ICOL  I    IJ  J1RW J2RW J12R LROW J1CL J2CL J12C LCOL JTOT    VEE
   3    1    1    1    1    2   18    0    0    0   20   20   1.3350521E+00
   4    1    2    1    1    2   20    0    0    0   20   20  -1.1190336E+00
   5    1    3    1    1    2   22    0    0    0   20   20   1.4019272E+00
   6    2    4    0    2    2   18    1    1    0   20   20  -1.1941069E+00
   7    2    5    0    2    2   20    1    1    0   20   20   1.0008941E+00
   8    2    6    0    2    2   22    1    1    0   20   20  -1.2539218E+00
   6    3    7    0    2    2   18    1    1    2   18   20   6.5271395E-01
   7    3    8    0    2    2   20    1    1    2   18   20  -9.0849966E-01
   6    4    9    0    2    2   18    1    1    2   20   20  -9.0849966E-01
   7    4   10    0    2    2   20    1    1    2   20   20  -7.0204698E-01
   8    4   11    0    2    2   22    1    1    2   20   20  -8.2406010E-01
   7    5   12    0    2    2   20    1    1    2   22   20  -8.2406010E-01
   8    5   13    0    2    2   22    1    1    2   22   20   7.5643981E-01
 LAMBDA= 1, NUMBER OF NONZERO MATRIX ELEMENTS =    13

 **  LIST OF CHANNELS:
   N   J1  J2  J12  L   E-(CM-1)   E-(HARTREE)

   1    0   0   0   20    0.000    0.00000E+00
   2    1   1   0   20    3.187    1.45220E-05
   3    1   1   2   18    3.187    1.45220E-05
   4    1   1   2   20    3.187    1.45220E-05
   5    1   1   2   22    3.187    1.45220E-05
   6    0   2   2   18    4.771    2.17392E-05
   7    0   2   2   20    4.771    2.17392E-05
   8    0   2   2   22    4.771    2.17392E-05
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:01 / CPU 00:00:00 / MP RATIO    0.3 %
      CURRENT DATE:  Jan 24 12:01:36 2023
 ===============================================================================
